New version: BandedMatrices v1.6.0

JuliaRegistrator opened the pull request:
- Registering package: BandedMatrices
- Repository: https://github.com/JuliaLinearAlgebra/BandedMatrices.jl
- Created by: @jishnub
- Version: v1.6.0
- Commit: a44ab1d220b351984e6f5d87db6d73ebfa7103bf
- Reviewed by: @jishnub
- Reference: https://github.com/JuliaLinearAlgebra/BandedMatrices.jl/commit/a44ab1d220b351984e6f5d87db6d73ebfa7103bf#commitcomment-139597664
- Description: A Julia package for representing banded matrices
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d3434cf970bcfdb29f2ccb9d4d1453db48f6a42a5a133f1e4f938419a7c4630b0016a7d5d2b7f1d44b857265b6d91d99fb7427afdffbb04474c4e1eb999714e7619d0a2c0e80617da6761017183a51d70a7843533a966028241249060389fc941f1224e0f5691639fc78be96f46f39101f5bc87f2f2c320186a6a5b63c8096b94840ca841099dccd1b9d06cecc2d7c7ab848af0dfcf4cbe29874f89d98b9680f7389c8ebfaa7a4e91f34bded21612b8956061fcf9e9f4dbd80e14336ec8e85c7 -->

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@jishnub Can you re-trigger Registrator? This will fix the merge conflict.

JuliaRegistrator committed:
New version: BandedMatrices v1.6.0

UUID: aae01518-5342-5314-be14-df237901396f
Repo: https://github.com/JuliaLinearAlgebra/BandedMatrices.jl.git
Tree: fe7ed7b33416db994f706f66e31bbd1b378f71b7

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
